Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -1879,11 +1880,11 @@ FOR_EACH_WASM_MEMORY_STORE_OP(CREATE_CASE)
             WASM_PARSER_FAIL_IF(!args.tryReserveCapacity(argc), "can't allocate enough memory for array.new_fixed ", argc, " values");
```

Comment:
This is just reserving a capacity. So args.size() is still 0. We need to call `args.resize(...)` to make the size matching against the arguments size.

Reply:
Also, since `args` is definitely empty before this, we can use `tryReserveInitialCapacity` instead.